Sources/CryptoSwift/Rabbit.swift

```diff
@@ -64,12 +64,12 @@ final public class Rabbit {
         
         // Key divided into 8 subkeys
         var k = [UInt32](count: 8, repeatedValue: 0)
-        for var j = 0; j < 8; j += 1 {
+        for j in 0 ..< 8 {
```

Comment:
Speaking of Xcode problems, would you mind if I submitted a pull request to add `*.xcscmblueprint` to the `.gitignore`?

Whenever I work on this project, Xcode insists on creating the file `CryptoSwift.xcodeproj/project.xcworkspace/xcshareddata/CryptoSwift.xcscmblueprint` which is not in the repo and which I'm constantly deleting.